pages/osx/emond.md

```diff
@@ -0,0 +1,17 @@
+# emond
+
+> Event Monitor service that accepts events from various services, runs them through a simple rules engine, and takes action.
+> The actions can run commands, send email, or SMS messages.
+> It should not be invoked manually.
+
+- Start the daemon:
+
+`emond`
+
+- Specify rules by giving a file or directory:
```

Comment:
I don't understand this example description. Which kind of rules?
```suggestion
- Specify rules by giving a file or directory:
```

Reply:
I updated it to this:
```
- Specify rules for emond to process by giving a path to a file or directory:
```
Emond accepts events, checks the events against a set of rules, and if there's a match, takes the specified action in the corresponding rule.  is the updated description more clear?